include/iso_alloc_internal.h

```diff
@@ -169,6 +169,19 @@ using namespace std;
 #define LIKELY(x) __builtin_expect(!!(x), 1)
 #define UNLIKELY(x) __builtin_expect(!!(x), 0)
 
+#if ALLOC_SANITY
+#if MEMCPY_SANITY
```

Comment:
Maybe skip a line, to make it easier to read, like:

```C
#if ALLOC_SANITY

#if MEMCPY_SANITY
#define iso_memcpy _iso_memcpy
#define iso_memset _iso_memset
#else
#define iso_memcpy __builtin_memcpy
#define iso_memset __builtin_memset
#endif

#else
#define iso_memcpy memcpy
#define iso_memset memset
#endif

Reply:
This set of defines is going to get tricky because I'd like to be able to enable `MEMCPY_SANITY` without `ALLOC_SANITY` being enabled. There should not be a dependency between the two.